inference-engine/src/low_precision_transformations/src/mat_mul.cpp

```diff
@@ -186,15 +180,22 @@ bool MatMulTransformation::canBeTransformed(const TransformationContext& context
         }
     }
 
-    const auto dequantization2 = ngraph::pass::low_precision::NetworkHelper::getDequantization(layer, 1);
+    const auto dequantization2 = NetworkHelper::getDequantization(layer, 1);
     if (!dequantization2.empty()) {
-        if ((updatePrecisions && !dequantization2.isLowPrecision()) || (dequantization2.subtract)) {
+        if ((updatePrecisions && !dequantization2.isLowPrecision())) {
             return false;
         }
 
-        const auto mulConst = as_type_ptr<opset1::Constant>(dequantization2.multiply->get_input_node_shared_ptr(1));
-        if (!NetworkHelper::isScalarLike(mulConst)) {
-            const auto constantShape = mulConst->get_shape();
+        if (dequantization2.subtract) {
+            std::shared_ptr<Node> roundedConst = NetworkHelper::round(dequantization2.subtractConstant, dequantization2.data.get_element_type());
+            roundedConst = NetworkHelper::toScalarIfPossible(roundedConst);
```

Comment:
remove this line, please: `roundedConst = NetworkHelper::toScalarIfPossible(roundedConst);`

Reply:
Fixed, thanks